hare/eligibility/oracle.go

```diff
@@ -13,13 +14,23 @@ import (
 )
 
 const k = types.LayerID(25) // the confidence interval // TODO: read from config
```

Comment:
move this to the configuration

Reply:
also, rename to something more meaningful...

Reply:
Fixed